OpenSim/Common/DataTable.h

```diff
@@ -1350,26 +1355,42 @@ class DataTable_ : public AbstractDataTable {
 
     // Split element into constituent components and append the components to
     // the given vector. For example Vec3 has 3 components.
```

Comment:
Can you add to the documentation that this function will write `N` elements starting from `begin` and does not necessarily write all the way to `end`?